deploy/examples/image_registry.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: app.kiegroup.org/v1
+kind: KieApp
+metadata:
+  name: image-registry
+spec:
+  environment: rhpam-trial
+  imageRegistry:
+    insecure: true
+    registry: registry.example.com:5000
```

Comment:
See our validation [unit test,](https://github.com/kiegroup/kie-cloud-operator/blob/master/pkg/controller/kieapp/test/crd_validation_test.go#L26). I think we should extend it, or add another function for the examples folder.

Reply:
i already added it with a prior PR...
https://github.com/kiegroup/kie-cloud-operator/blob/master/pkg/controller/kieapp/test/crd_validation_test.go#L38